ci/flaky_test/process_xml.py

```diff
@@ -97,25 +97,27 @@ def getGitInfo(CI_TARGET):
     os.environ["TMP_OUTPUT_PROCESS_XML"] = os.getenv("TEST_TMPDIR") + "/tmp_output_process_xml.txt"
   else:
     print("set the env variables first")
-    sys.exit(0)
+    sys.exit(1)
```

Comment:
Not sure if we want to exit 1 here or leave it as 0? I guess it is ok if someone didn't set variables, means they dont want to run the script